public/components/notebooks/components/note_table.tsx

```diff
@@ -367,10 +380,9 @@ export function NoteTable(props: NoteTableProps) {
                 <EuiSpacer size="m" />
                 <EuiFlexGroup justifyContent="center">
                   <EuiFlexItem grow={false}>
-                    <EuiButton
+                    <EuiButton fill href="#/notebooks/create"
```

Comment:
Let's revert this fill as it was previously. Usually, a page should only have 1 primary fill button to implicitly tell the user that this is the main button to click on.

Reply:
I see... Fixed in [cc0763c](https://github.com/opensearch-project/dashboards-observability/pull/152/commits/cc0763cb6612d999418849812427d33a7ca931f8)